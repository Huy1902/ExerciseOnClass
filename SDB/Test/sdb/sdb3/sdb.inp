20
18
6
11
14
5
17
5
14
15
14
7
8
14
14
19
5
1
19
9
2
