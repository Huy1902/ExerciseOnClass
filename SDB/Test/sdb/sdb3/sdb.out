10
18
6
11
17
15
7
8
1
9
2
